lib/matplotlib/backends/_backend_tk.py

```diff
@@ -234,10 +233,13 @@ def filter_destroy(event):
         self._rubberband_rect_white = None
 
     def _update_device_pixel_ratio(self, event=None):
-        # Tk gives scaling with respect to 72 DPI, but Windows screens are
-        # scaled vs 96 dpi, and pixel ratio settings are given in whole
-        # percentages, so round to 2 digits.
-        ratio = round(self._tkcanvas.tk.call('tk', 'scaling') / (96 / 72), 2)
+        if sys.platform == 'win32':
+            # Tk gives scaling with respect to 72 DPI, but Windows screens are
+            # scaled vs 96 dpi, and pixel ratio settings are given in whole
+            # percentages, so round to 2 digits.
+            ratio = round(self._tkcanvas.tk.call('tk', 'scaling') / (96 / 72), 2)
+        else:
+            ratio = self._tkcanvas.winfo_fpixels('1i') / 96
```

Comment:
```suggestion
            ratio = self._tkcanvas.winfo_fpixels('1i') / 72
```

Shouldn't this be 72?

Reply:
I'm not sure, but it looks correct visually.

Reply:
I found this, which may or may not explain 96. 🤷‍♂️

https://wiki.tcl-lang.org/page/tk+scaling

> Note that the [tk scaling] value set at startup by Tk seems to use a reference screen having a resolution density of 96 pixels/inch (this value seems to come from the OS). If you monitor has 72 pixels/inch, you are off by a factor of 0.75 (see this discussion on c.l.t ).